### PR TITLE
BNF 147 160 39 9 EMML 1292

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record Add 160, minor corrections for others.

### CODE REVIEW

File: ParisBNF/abb/BNFabb160.xml [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno facs="https://archivesetmanuscrits.bnf.fr/ark:/12148/cc1056302">BnF Éthiopien d'Abbadie 160</idno>
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+
+                             <msItem xml:id="ms_i1">
+                               <locus from="1r" to="73"/>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb160.xml:37]: is 73 page or folium? if page these must be recalculated

File: ParisBNF/abb/BNFabb160.xml [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+                                         <height>95</height>
+                                         <width>80</width>
+                                      </dimensions>
+                                   </extent>
+
+                                   <foliation>The manuscript is paginated, the page numbers
+                                     are written in the outer corner of the upper margin.
+                                     Leaves at the beginning and the end of the volume with no page numbers have been paginated as
+                                    <locus target="#i #ii #iii #v #vi #vii"/>
+                                     and <locus target="#74 #75 #76 #77 #78 #79"/>.
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb160.xml:159]: locus only takes folio numbers, not page numbers, make sure you recalculate

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb160.xml [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+
+                             <msItem xml:id="ms_i1">
+                               <locus from="1r" to="73"/>
+                               <title type="complete">Sawāsǝw-treatise on lexicon and grammar</title>
+                               <textLang mainLang="gez"/>
+
+                               <msItem xml:id="ms_i1.1">
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb160.xml:41]: I think it would make sense to add everywhere to textLang `@otherLangs="am"`

**Review by thea-m — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb160.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb160.xml:37]: ok

File: ParisBNF/abb/BNFabb160.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb160.xml:41]: ok

File: ParisBNF/abb/BNFabb160.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb160.xml:159]: ok

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**